File: Exercises/T3/src/java/Cargos/Gerente.java
package Exercises.T3.src.java.Cargos;
import Exercises.T3.src.java.GestaoABC.Funcionaria;

public class Gerente extends Funcionaria {
  public Gerente(String nome, String CPF, double salarioBase) {
    this.nome = nome;
    this.CPF = CPF;
    this.salarioBase = salarioBase;
  };

  public double calculaSalario() {
    return this.salarioBase * 2;
  };

  @Override
  public String toString() {
    String str = String.format("Cargo: Gerente%n");
    str += this.superToString();
    return str;
  }
}

File: Exercises/T3/src/python/Cargos/Gerente.py
from GestaoABC.Funcionaria import Funcionaria

class Gerente(Funcionaria):
    def __init__(self, nome: str, CPF: str, salarioBase: float) -> None:
        super().__init__(nome, CPF, salarioBase)
    
    def calculaSalario(self) -> float:
        return self.salarioBase * 2
    
    def __str__(self) -> str:
        return "Cargo: Gerente\n" + super().__str__()